source/hardware/controllers/yaeltex_minimixxx.rst

```diff
@@ -0,0 +1,328 @@
+.. _yaeltex-minimixxx:
+
+Yaeltex MiniMixxx
+=================
+
+-  `Manufacturer’s product page <https://yaeltex.com/product/minimixxx/>`__
+
+.. sectionauthor::
+  Owen Williams <owilliams at mixxx.org>
+
+.. versionadded:: 2.4.0
+
+The Yaeltex MiniMixxx is a compact, 2-deck DJ controller designed exclusively
+for Mixxx. Unlike most small form-factor DJ controllers which sacrifice
+important functions, the MiniMixxx provides access to all of the critical parts
+of DJing: full size EQ and quick FX knobs, pitch sliders, channel faders and
+track gain, playback controls including sync nudging, looping, an FX bank, 4
+hotcues per deck, 8 samplers, sync lock, keylock, library navigation, and master
+output settings. Button mode layers are intelligently laid out and road-tested
+for maximum flexibility during performance, with bold RGB colors indicating
+current modes. The visualization of channel volume and virtual record spinning
+indicators makes the MiniMixxx an eye-catching wonder in the DJ booth.
+
+The MiniMixxx can run on USB bus power. Using a separate power supply increases
+the brightness of the LEDs. The MiniMixxx does not have a built-in sound card.
+
+Compatibility
+-------------
+
+Controller
+~~~~~~~~~~
+
+The MiniMixxx is a USB class compliant MIDI device.  To work correctly with this
+mapping, you need to load the custom Yaeltex layout file onto your controller:
+
+1. Download the :download:`MiniMixxx Layout Configuration<../../_static/controllers/MiniMixxx.ytx>`.
+1. Go to `Yaeltex Kilowhat <https://kilowhat.yaeltex.com/>`_ configuration site.
+1. Select your controller and click "Load From Desktop"
+1. Select the downloaded layout file.
+1. Click Send to Device.
+
+Inside Mixxx, open the preferences and select the Yaeltex Minimixxx from the list of controller presets.
+
+Mapping description
+-------------------
+
+.. figure:: ../../_static/controllers/yaeltex_minimixxx.jpg
```

Comment:
Do we have a permission to use this photo, or is it your own work?

Reply:
I was working with the manufacturer to design the controller and they are ok with this.